specification/cognitiveservices/data-plane/FormRecognizer/preview/v2.1-preview.2/FormRecognizer.json

```diff
@@ -0,0 +1,2123 @@
+            "pixel",
+            "inch"
+          ],
+          "x-ms-enum": {
+            "name": "LengthUnit",
+            "modelAsString": false
+          },
+          "x-nullable": false
+        },
+        "language": {
+          "description": "The detected language on the page overall.",
+          "$ref": "#/definitions/Language"
+        },
+        "lines": {
+          "description": "When includeTextDetails is set to true, a list of recognized text lines. The maximum number of lines returned is 300 per page. The lines are sorted top to bottom, left to right, although in certain cases proximity is treated with higher priority. As the sorting order depends on the detected text, it may change across images and OCR version updates. Thus, business logic should be built upon the actual line location instead of order.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/TextLine"
+          }
+        },
+        "selectionMarks": {
+          "description": "List of selection marks extracted from the page.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/SelectionMark"
+          }
+        }
+      }
+    },
+    "Language": {
+      "description": "Language code",
+      "type": "string",
+      "enum": [
+        "en",
+        "es"
+      ],
+      "x-ms-enum": {
+        "name": "Language",
+        "modelAsString": true
+      },
+      "x-nullable": false
+    },
+    "TextLine": {
+      "description": "An object representing an extracted text line.",
+      "type": "object",
+      "required": [
+        "text",
+        "boundingBox",
+        "words"
+      ],
+      "properties": {
+        "text": {
+          "description": "The text content of the line.",
+          "type": "string"
+        },
+        "boundingBox": {
+          "description": "Bounding box of an extracted line.",
+          "$ref": "#/definitions/BoundingBox"
+        },
+        "language": {
+          "description": "The detected language of this line, if different from the overall page language.",
+          "$ref": "#/definitions/Language"
+        },
+        "words": {
+          "description": "List of words in the text line.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/TextWord"
+          }
+        }
+      }
+    },
+    "TextWord": {
+      "description": "An object representing a word.",
+      "type": "object",
+      "required": [
+        "boundingBox",
+        "text"
+      ],
+      "properties": {
+        "text": {
+          "description": "The text content of the word.",
+          "type": "string"
+        },
+        "boundingBox": {
+          "description": "Bounding box of an extracted word.",
+          "$ref": "#/definitions/BoundingBox"
+        },
+        "confidence": {
+          "description": "Confidence value.",
+          "$ref": "#/definitions/Confidence"
+        },
+        "appearance": {
```

Comment:
@bojunehsu confirmed that `appearance` should be moved to TextLine

Reply:
done